generator/generator.yml

```diff
@@ -635,7 +586,6 @@ modules:
 # http://download2.mikrotik.com/Mikrotik.mib
   mikrotik:
     walk:
-      - interfaces
       - ifMIB
```

Comment:
I wonder if we should remove this as well, as it's covering both `interfaces` and `ifXTable`.

Reply:
`ifMIB` translates to `1.3.6.1.2.1.31` whereas `interfaces` is `1.3.6.1.2.1.2` and `ifXTable` `1.3.6.1.2.1.31.1.1` is a subset of `ifMIB` 
We'd be missing `ifStackTable`, `ifTestTable`, `ifRcvAddressTable`, `ifTableLastChange` and `ifStackLastChange` but i doubt that these are of any value

Reply:
it took me some time to get hands on a mikrotik device to test the changes.
removed also the ifMIB as the few other OIDs are not really useful